client/network/src/protocol/message.rs

```diff
@@ -147,7 +144,7 @@ pub mod generic {
 		/// Block announce.
 		BlockAnnounce(BlockAnnounce<Header>),
 		/// Transactions.
-		Transactions(Transactions<Extrinsic>),
+		Transactions,
```

Comment:
This seems to be not used. May be we should remove `Transactions` completely and just put `#[codec(index = 6)]` annotation to `Consensus` message with a comment?

Reply:
@bkchr told me to leave it as is in the previous PR. If removing it touches consensus in any way, I'd be happier if someone else made the choice whether to keep it or not.

Reply:
I'm not sure too, @bkchr should know better.

Reply:
@dmitry-markin @bkchr I replaced `Transaction` with `#[codec(index = 6)]` and this "new" tx protocol was able to talk with a Polkadot node running latest changes from master so it should be an OK change. Let me know what you think

Reply:
Sounds good to me! But please also add a comment why we have the index there.